Kristian-Repizo/tarea5/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Puzzle Game</title>
+    <link rel="stylesheet" href="/css/style.css">
```

Comment:
@Kristian-Repizo , you should use relative paths

`<link rel="stylesheet" href="/css/style.css">`